tests/main_test.go

```diff
@@ -131,6 +133,64 @@ func pebbleRun(t *testing.T, pebbleDir string, args ...string) (stdoutCh chan se
 	return stdoutCh, stderrCh
 }
 
+// pebbleEnter runs `pebble enter` with optional arguments
+// and returns two channels for standard output and standard error.
+func pebbleEnter(t *testing.T, pebbleDir string, args ...string) (stdoutCh chan string, stderrCh chan string) {
```

Comment:
I saw your comment about trying to merge this with `pebbleRun` with an interface -- but I think it's a lot simpler than that. We can basically use what you have as `pebbleEnter` for both run and enter (I've called it `pebbleDaemon`), so it returns lines. And then change `waitForLog` to use `servicelog.Parse` on each line.

https://gist.github.com/benhoyt/4cfe74acaa366cdc8a4fc7f5e3a1d9eb